### DISCUSSION TITLE
Regarding returns in BackTestStats

### DISCUSSION THREAD

**Discussion by sahibnoor1996** [AI4Finance-Foundation/FinRL#764] [Q&A]
I would like to ask you about the returns (Deep Reinforcement Learning for Stock Trading from Scratch: Multiple Stock Trading Using Ensemble Strategy). It showed an Annual return  of 0.079474 in the end. Can you please clarify whether this means 0.07 percent or it means 7 percent returns yearly?

**Comment by zhumingpassional** [comment]
7 percent
